src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -1,39 +1,29 @@
 @echo off
 setlocal
 
-set PROJECT_DIR=%~1
-set DOTNET_CMD=%~2
-set TOOLRUNTIME_DIR=%~3
-IF [%BUILDTOOLS_TARGET_RUNTIME%]==[] set BUILDTOOLS_TARGET_RUNTIME=win7-x64
+set DOTNET_CMD=%~1
+set TOOLRUNTIME_DIR=%~2
 set BUILDTOOLS_PACKAGE_DIR=%~dp0
-set PORTABLETARGETS_VERSION=0.1.1-dev
 set ROSLYNCOMPILERS_VERSION=1.2.0-beta1-20160202-02
-set MSBUILD_CONTENT_JSON={"dependencies": {"Microsoft.Portable.Targets": "%PORTABLETARGETS_VERSION%", "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%"},"frameworks": {"dnxcore50": {},"net46": {}}}
+set NuProj_Version=0.10.4-beta-gf7fc34e7d8
+set MSBUILD_CONTENT_JSON={"dependencies": { "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%", "NuProj": "%NuProj_Version%" },"frameworks": {"dnxcore50": {},"net46": {}}}
```

Comment:
These two packages cannot be published using dotnet cli because their format is:
```
<PackageRoot>
    -  Tools
         - ......
```
And dotnet cli doesn't support publishing tools folder. To fix this, we would need the packages to follow the contentFiles convention.